lib/puppet/provider/user/directoryservice.rb

```diff
@@ -619,13 +619,4 @@ def set_salted_pbkdf2(users_plist, shadow_hash_data, field, value)
   def write_users_plist_to_disk(users_plist)
     Puppet::Util::Plist.write_plist_file(users_plist, "#{users_plist_dir}/#{@resource.name}.plist", :binary)
   end
-
-  # This is a simple wrapper method for writing values to a file.
-  def write_to_file(filename, value)
```

Comment:
That context is helpful. I think I'd still be +1 to deprecating even in lieu of it. It'll only be around for another minor version, and I can sleep better the night of the 4.9.0 release 😄